doc/admin/pool_operations.md

```diff
@@ -296,11 +300,11 @@ The example below shows a rebuild in progress and NVMe space allocated.
     Pool space info:
     - Target(VOS) count:56
     - SCM:
-        Total size: 30064771072
-        Free: 29885237632, min:493096384, max:536869696, mean:533664957
+        Total size: 28GB
+        Free: 28GB, min:470MB, max:512MB, mean:509MB
     - NVMe:
-        Total size: 60129542144
-        Free: 29885237632, min:493096384, max:536869696, mean:533664957
+        Total size: 56GB
+        Free: 28GB, min:470MB, max:512MB, mean:509MB
```

Comment:
Not a showstopper, but It's perhaps a little unusual for this pool to show the same quantities of free/min/max/mean for both SCM and NVMe. Is that easy enough to replace with something different? Also there is the base 2 (SCM) versus base 10 (NVMe) distinction that (if one were translating from the raw "total size" value in the line removed - which technically doesn't matter in the final form of the doc) here would probably be 60GB in the output.

Reply:
agreed,  I just translated the values from the raw bytes that were provided in the original text. someone obviously just copied these values from above.